docs/wallet/wallet_creation.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 1
+---
+
+# How to create a wallet from the beginning by using the wallet API
+1. Follow this link: [Camino Wallet](https://wallet.camino.foundation/)
+
+    **TIP:** The language can be changed in the upper right corner.
+
+2. Choose network  
+    Options:  
+   - Camino Mainnet
+   - Columbus Testnet
+
+   <img alt="image" src="/img/wallet/create_wallet1.png" height="180"  width="550"/> <img alt="image" height="180" src="/img/wallet/create_wallet2.png" width="310" align="right"/>   <br />
+
+
+3. Click on the top button “create new wallet”.    <br /> 
+   <img alt="image" src="/img/wallet/create_wallet3.png"/>
+
+
+4. Now you have to generate a key phrase. This can be done by clicking on the button “generate key phrase”.  
+   <img alt="image" height="250" src="/img/wallet/create_wallet4.png" width="340"/>  <img alt="image" height="250" src="/img/wallet/create_wallet5.png" width="538"/>
+
+   **TIP:** Once you clicked “generate key phrase” the mnemonic key phrase will pop up. If then you wish to get a different mnemonic key phrase, you can click on “regenerate” and a new phrase will pop up.
+
+5. Copy the phrase and store it at a secure location.
+
+    **TIP:** Store this key phrase in a secure location. Anyone with this key phrase can access your camino wallet. There is no way to recover lost key phrases.
+
+6. Once you secured your key phrase, tick the box.    <br />
+<img alt="image" height="316" src="/img/wallet/create_wallet6.png" width="306"/>
+
+7. Click “access wallet”.
+
+8. Fill the gaps from your saved mnemonic phrase and press the “Verify” button.   <br />
+   <img alt="image" height="250" src="/img/wallet/create_wallet7.png" width="538" align="top"/> <img alt="image" height="500" src="/img/wallet/create_wallet8.png" width="350"/>
+
+9. When the mnemonic phrase is correct, the green “check” will appear.
+
+10. Now you can access your wallet by clicking “access wallet”.
+    ![image](/img/wallet/create_wallet9.png)
+
+
+
+## How to save your wallet login
+After you entered the wallet, you will have the possibility to add a username and password to your wallet. Option: This is not an obligatory option.
+
+1. Click on the button “Save account”  in the bottom left corner:   <br />
+   ![image](/img/wallet/create_wallet10.png)
+
+2. Fill all required fields. Make sure that your password contains upper and lowercase  <img alt="image" height="400" src="/img/wallet/create_wallet11.png" width="260" align="right"/>    <br />
+   letters + numbers. The length of the password has to be more than 8 symbols.
+3. Press the “SAVE” button.
+
+4. Your wallet is now saved with the username and password you chose.
+   <br />
```

Comment:
Yes, it is because of images with html formatting....i agree that it looks strange, but i didn't find any way to place the text and pictures normally

Reply:
I think its good to have a step on step guide for the end user